Makefile

```diff
@@ -0,0 +1,180 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############
+# Dev Helpers #
+###############
+SHELL := /bin/bash
+GOBIN ?= $$PWD/bin
+GOOS ?= linux
+define build
+    mkdir -p $(GOBIN) && \
+		GOOS=$(GOOS) \
+		GOARCH=amd64 \
+		CGO_ENABLED=0 \
+		GOBIN=$(GOBIN) \
+		GO111MODULE=on \
+		go install -v \
+		-gcflags='-e' \
+		$(1)
+endef
+
+
+############
+# Operator #
+############
+olocal:
+	@./dev/operator_local.sh
+
+ostop:
+	@.kubectl -n=cortex delete --ignore-not-found=true deployment operator
+
+opudate:
+	@./cortex.sh -c=./dev/config/cortex.sh update operator
+
+oinstall:
+	@./cortex.sh -c=./dev/config/cortex.sh install operator
+
+ouninstall:
+	@./cortex.sh -c=./dev/config/cortex.sh uninstall operator
+
+
+#######
+# EKS #
+#######
+eks-up:
+	@./dev/eks.sh start
+	$(MAKE) oinstall
+
+eks-down:
+	$(MAKE) ouninstall || true
+	@./dev/eks.sh stop
+
+eks-set:
+	@./dev/eks.sh set
+
+########
+# KOPS #
+########
+kops-up:
+	@./dev/kops.sh start
+	@./cortex.sh -c=dev/config/cortex.sh install operator 
+	@kubectl -n=cortex delete --ignore-not-found=true deployment operator
```

Comment:
This probably shouldn't be here (since it's not in `eks-up`). What do you think about a command like `kops-up-dev` which calls `kops-up` and `ostop`? (could do the same for eks)